src/Data/Attoparsec/Interpreter.hs

```diff
@@ -146,9 +146,9 @@ getInterpreterArgs file = do
 
     decodeError e =
       case e of
-        ParseError ctxs _ (Position line col) ->
+        ParseError ctxs _ (Position line col offset) ->
           if null ctxs
           then "Parse error"
           else ("Expecting " ++ intercalate " or " ctxs)
-          ++ " at line " ++ show line ++ ", column " ++ show col
+          ++ " at line " ++ show line ++ ", column " ++ show col ++ ", offset " ++ show offset
```

Comment:
Fairly slow as usual but why is the offset useful in this context?

Reply:
I am not sure about this either, could remove it if you think it's not relevant :)

Reply:
Well, do you have an example of the output? Maybe it's useful?